src/core/sys/posix/aio.d

```diff
@@ -0,0 +1,66 @@
+/**
+ * D header file to interface with the Linux aio API (http://man7.org/linux/man-pages/man7/aio.7.html).
+ * Available since Linux 2.6
```

Comment:
So this should be in core/sys/linux then, not posix.

Reply:
Actually no, there are two instance of AIO, the POSIX aio (aio_* family) and Linux libaio.
I believe this is in the correct place.

Reply:
This is POSIX AIO interface except for aio_init I think.